lib/explorer/backend/data_frame.ex

```diff
@@ -41,7 +41,7 @@ defmodule Explorer.Backend.DataFrame do
               result(String.t())
 
   @callback from_parquet(filename :: String.t()) :: result(df)
-  @callback to_parquet(df, filename :: String.t()) :: result(String.t())
+  @callback to_parquet(df, filename :: String.t(), compression :: atom()) :: result(String.t())
```

Comment:
For what I could understand, this is nullable, right?
```suggestion
  @callback to_parquet(df, filename :: String.t(), compression :: atom() | nil) :: result(String.t())
```

I know `nil` is an atom, but it's good to make it explicit. WDYT?

Reply:
Good point :)